File: AndroidOfficiel/app/src/main/java/Model/Status.java
package Model;

public class Status {
    private int id;
    private String state;

    public Status(int id, String state) {
        this.id = id;
        this.state = state;
    }

    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }

    public String getState() {
        return state;
    }
    public void setState(String state) {
        this.state = state;
    }
}

File: AnimaLost/AnimaLost/Model/Status.cs
using System.Collections.Generic;

namespace AnimaLost.Model
{
    public class Status
    {
        public Status()
        {
            Announcement = new HashSet<Announcement>();
        }

        public int Id { get; set; }
        public string State { get; set; }

        public ICollection<Announcement> Announcement { get; set; }
    }
}